system/jlib/jstats.h

```diff
@@ -874,6 +875,37 @@ void setStat(CRuntimeStatisticCollection & stats, INTERFACE * source, StatisticK
 template <class INTERFACE>
 void setStat(CRuntimeStatisticCollection & stats, const Shared<INTERFACE> & source, StatisticKind kind) { setStat(stats, source.get(), kind); }
 
+template <class INTERFACE>
+void mergeRemappedStats(CRuntimeStatisticCollection & stats, INTERFACE * source, const StatisticsMapping & mapping, const std::map<StatisticKind, StatisticKind> & remaps)
```

Comment:
I think worth defining the remap type as a typedef:
typedef std::map<StatisticKind, StatisticKind> StatKindMap;
(and using elsewhere)